packages/create-llama/e2e/basic.spec.ts

```diff
@@ -48,6 +48,7 @@ for (const templateType of templateTypes) {
           let cwd: string;
           let name: string;
           let cps: ChildProcess[] = [];
+          const postInstallAction = "dependencies";
```

Comment:
why not using `runApp`?

```suggestion
          const postInstallAction = "runApp";
```

Reply:
then we could simplify the test code and also cover the test for runapp

Reply:
runApp action will block the process until it gets an exit signal, while the testing will start the app and return child processes and kill them afterward. It'll be better to refactor the runApp as a non-blocking function to improve the flexibility, do you want me to implement it in this PR?